tests/decode.cpp

```diff
@@ -103,6 +103,13 @@ int main(int argc, char** argv)
     }
 
     status = decoder->start(&configBuffer);
+
+    formatInfo = decoder->getFormatInfo();
+    if (formatInfo->width && formatInfo->height) {
+        if (!output->setVideoSize(formatInfo->width, formatInfo->height))
```

Comment:
could we put setVideoSize to this if?
if (input->getWidth() && input->getHeight()) {
        configBuffer.surfaceWidth = input->getWidth();
        configBuffer.surfaceHeight = input->getHeight();
    }